modules/nf-core/custom/tabulartogseagct/main.nf

```diff
@@ -0,0 +1,33 @@
+process CUSTOM_TABULARTOGSEAGCT {
+    tag "$meta.id"
+    label 'process_single'
+
```

Comment:
I would still add a container here for reproducibility. (It will work in most cases, but we gotta think of those edge cases :) )